libvmi/os/windows/windows.h

```diff
@@ -23,6 +23,10 @@
 
 #include "private.h"
 
+#define LOWEST_KERNEL_ADDRESS_64BIT 0xffff800000000000
+#define LOWEST_KERNEL_ADDRESS_32BIT 0x80000000
```

Comment:
The problem with this on 32bit machines is that its configurable. You can define at boot time what the boundary should be, so while most of the time this its fine it can also easily break.